package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dummy-text",
+  "version": "0.0.1",
+  "description": "Generate custom dummy text via JSON dictionaries.",
+  "main": "index.js",
+  "module": "main.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CITguy/dummy-text.git"
+  },
+  "author": "Ryan Johnson (CITguy)",
+  "license": "GPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/CITguy/dummy-text/issues"
+  },
+  "homepage": "https://github.com/CITguy/dummy-text#readme",
+  "dependencies": {
+    "esm": "^3.2.25"
+  },
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^8.1.3"
+  }
+}
```

Comment:
- [ ] `"type": "module"`
- [ ] `"exports": { ".": "lib/index.mjs" }`
- [ ] add metadata for minimum Node version